templates/rag-lancedb/pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[tool.poetry]
+name = "rag-lancedb"
+version = "0.0.1"
+description = "lancedb as vectordb integration"
+authors = [ "akash desai  <aksdesai1998@gmail.com>",]
+license = "MIT"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
+langchain = "^0.0.330"
+lancedb = "^0.3.3"
+openai = "^0.28.1"
+tiktoken = "^0.5.1"
+
+[tool.poetry.group.docs.dependencies]
+langchain = { path = "libs/langchain/", develop = true }
```

Comment:
Would be better to tie to a specific version so people can use it out of the box when they start it up!

Reply:
okay so should i kept same openai,tiktoken version as other projects